phpunit/fixtures/long-content.html

```diff
@@ -66,7 +66,7 @@ <h2>Visual Editing</h2>
 <p>Blocks can be anything you need. For instance, you may want to add a subdued quote as part of the composition of your text, or you may prefer to display a giant stylized one. All of these options are available in the inserter.</p>
 <!-- /wp:paragraph -->
 <!-- wp:gallery {"columns":2} -->
-<div class="wp-block-gallery alignnone columns-2 is-cropped">
```

Comment:
When using `align` there is no `alignnone` - let's remove it to avoid having it loaded as a custom CSS class.